tests/integration_tests/test_cli.py

```diff
@@ -393,7 +393,7 @@ def test_that_setenv_sets_environment_variables_in_jobs(setenv_config):
             "sh",
             dedent(
                 """\
-                    #!/bin/bash
+                   #!/bin/bash
```

Comment:
This is The Fix ?

Reply:
That is why the test was broken. It failed on this instead of the expected failure..